Sources/SwiftInspectorVisitors/Tests/ClassVisitorSpec.swift

```diff
@@ -51,8 +51,8 @@ final class ClassVisitorSpec: QuickSpec {
 
             let classInfo = self.sut.classes.first
             expect(classInfo?.name) == "SomeClass"
-            expect(classInfo?.inheritsFromTypes) == []
-            expect(classInfo?.parentTypeName).to(beNil())
+            expect(classInfo?.inheritsFromTypes.map { $0.description }) == []
```

Comment:
In order to keep the test signatures the same (and also test `description`), I always inspect the `description` at the test sites. It isn't necessary on this line, but it's necessary in most spots. See below for how it's used, but I don't think these test files need detailed review (except where called out explicitly).